contrib/dokku_client.sh

```diff
@@ -98,9 +98,24 @@ if [[ ! -z $DOKKU_HOST ]]; then
       exit $?
     fi
 
-    [[ -z "$verb" ]] && verb=$1
-    shift
-    ssh "dokku@$DOKKU_HOST" "$verb" "$appname" "$@"
+    if [[ -z "$verb" ]]; then
+      if [[ ! "$1" =~ --* ]]; then
+        verb=$1
+        shift
+        args="$@"
+      else
+        long_args="--"
+        for arg in "$@";do
+          if [[ "$arg" =~ --* ]];then
+            long_args+=" $arg"
+            args=(${args[@]/$arg})
+          else
+             verb="$arg"
+          fi
+        done
+      fi
+    fi
+    ssh "dokku@$DOKKU_HOST" $long_args "$verb" "$appname" "$args"
```

Comment:
definitely open to a better way of doing this. its the client, so I kind of just hacked it for now